modules/configuring-firewall.adoc

```diff
@@ -146,7 +145,7 @@ When you add a site, such as `quay.io`, to your allowlist, do not add a wildcard
 |443, 80
 |Required both for your cluster token and to check if updates are available for the cluster.
 
-|`art-rhcos-ci.s3.amazonaws.com`
+|`rhcos.mirror.openshift.com`
```

Comment:
We'll want to keep both hostnames. People installing earlier 4.10.z will use the old hostname and those using newer 4.10.z will use the new hostname.

Reply:
Agree with sdodson.  Also `rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com` should be moved into this table here too.

Reply:
Thanks both, updated